wazimap_ng/points/views.py

```diff
@@ -77,6 +80,14 @@ def list(self, request, profile_id, profile_category_id=None, geography_code=Non
     def dispatch(self, *args, **kwargs):
         return super().dispatch(*args, **kwargs)
 
+def text_search(qs, search_terms):
```

Comment:
If the user searches by the theme /category name, would just filtering the content_search be enough? @goyal1092

Reply:
Nop @emre2038 It will not search by category name